lib/domains/character/CharacterFactory.tsx

```diff
@@ -33,64 +27,140 @@ import {
 
 export const CharacterFactory = {
   latestVersion: 3,
-  make(type: CharacterTemplates): ICharacter {
-    const newCharacter = {
+  async make(type: CharacterTemplates): Promise<ICharacter> {
+    const templateFunctions: Record<
+      CharacterTemplates,
+      () => Promise<ICharacter>
+    > = {
       /**
        * @author @RPDeshaies
        */
-      [CharacterTemplates.FateCondensed]: makeFateCondensedCharacter,
+      [CharacterTemplates.FateCondensed]: async () =>
+        makeFateCondensedCharacter(),
       /**
        * @author @RPDeshaies
        */
-      [CharacterTemplates.FateCore]: makeFateCoreCharacter,
+      [CharacterTemplates.FateCore]: async () => makeFateCoreCharacter(),
       /**
        * @author @RPDeshaies
        */
-      [CharacterTemplates.FateAccelerated]: makeFateAcceleratedCharacter,
+      [CharacterTemplates.FateAccelerated]: async () =>
+        makeFateAcceleratedCharacter(),
       /**
        * @author @RPDeshaies
        */
-      [CharacterTemplates.FateOfCthulhu]: makeFateOfCthulhuCharacter,
+      [CharacterTemplates.FateOfCthulhu]: async () =>
+        makeFateOfCthulhuCharacter(),
       /**
        * @author @LostInBrittany
        */
-      [CharacterTemplates.DresdenFilesAccelerated]: () => {
-        return this.makeFromJson(DrescendFilesAccelerated);
+      [CharacterTemplates.DresdenFilesAccelerated]: async () => {
+        const jsonData = await import(
+          "./character-templates/DrescendFilesAccelerated.json"
+        );
+        return this.makeFromJson(jsonData);
```

Comment:
templates are now loaded asynchronously to help with perfs and number of bytes downloaded when loading the character route